docs/design/generics/details.md

```diff
@@ -4991,7 +5976,11 @@ interface TotalOrder {
 The workaround for this restriction is to use a
 [blanket impl declaration](#blanket-impl-declarations) instead:
 
-```
+**FIXME: Is it sensible to have both a `require` and a blanket implementation?
+Does the blanket implementation satisfy the requirement so you only need to
+implement `TotalOrder`?**
```

Comment:
I think the `require` here has no effect, because the constraint is always satisfied by the blanket impl.